src/PKSim.Presentation/Presenters/Simulations/SimulationModelSelectionPresenter.cs

```diff
@@ -59,7 +59,7 @@ public IEnumerable<ModelConfiguration> AllModels()
       public void ModelSelectionChanging(ModelConfiguration newModelConfiguration)
       {
          ModelConfiguration = newModelConfiguration;
-         _modelPropertiesToUse = _modelPropertiesTask.Update(ModelProperties, _modelPropertiesTask.DefaultFor(newModelConfiguration, _originData), _originData);
+         _modelPropertiesToUse = _modelPropertiesTask.DefaultFor(newModelConfiguration, _originData);
```

Comment:
In a nutshell here - do we need to combine ModelProperties from old to new?

Reply:
this is what was us causing the problem. Creating new one from scratch should be good I believe
Every entry that was set by hand will be reset but this seems to be ok
Do we need the .Update method at all? if not, I would just remove it

Reply:
yeah, it's removed in this PR. I'll remove the tests for it and make the PR ready for review